packages/stream_video/lib/src/stream_video.dart

```diff
@@ -944,6 +946,24 @@ class StreamVideo extends Disposable {
 
     return Result.success(call);
   }
+
+  @internal
+  Future<Result<bool>> isAudioProcessingEnabled() async {
+    return await _options.audioProcessor?.isEnabled() ??
+        const Result.failure(VideoError(message: 'No audio processor found.'));
+  }
+
+  @internal
+  Future<Result<None>> setAudioProcessingEnabled(bool enabled) async {
+    return await _options.audioProcessor?.setEnabled(enabled) ??
+        const Result.failure(VideoError(message: 'No audio processor found.'));
+  }
+
+  Future<Result<bool>> deviceSupportsAdvancedAudioProcessing() async {
+    return await _options.audioProcessor
+            ?.deviceSupportsAdvancedAudioProcessing() ??
+        const Result.failure(VideoError(message: 'No audio processor found.'));
+  }
```

Comment:
related to the other comments, I would just return false here, instead of error.

Do we also need a method `deviceSupportsNoiseCanceling` or something similar?

Probably good to document here what exactly is `AdvancedAudioProcessing`.